stdlib/auth/src/main/ballerina/src/auth/Module.md

```diff
@@ -1,46 +1,3 @@
-## Module Overview
-
 This module provides default authentication provider configurations that can be extended to create new authentication providers.
 
-### Inbound Authentication Provider
```

Comment:
@Bhashinee shall we keep these technical details. I think it is worth to have them in API Docs